### PR TITLE
Update README and remove startup script

### PR DESCRIPTION
**Pull request by 1pretz1**
Updates README and removes startup script as part of the migration to GOV.UK Docker.

Trello:
https://trello.com/c/Psf7EzPB/204-move-away-from-startupsh

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,48 @@
...
-`bundle install`, then `./startup.sh`. `bundle exec rake` runs the tests.
+### Running the test suite
 
-Links for each local authority are available via data replication.
+```
+$ bundle exec rake
+```
 
-## Detailed documentation
+## Documentation
```
Comment (ChrisBAshton) [inline README.md]: We've been using:

```suggestion
### Further documentation
```

File: README.md [hunk 2]
```diff
@@ -1,49 +1,48 @@
...
+[govuk-docker]: https://github.com/alphagov/govuk-docker
+[docker-usage]: https://github.com/alphagov/govuk-docker#usage
 
-### Run locally
+Running the bare application without any test data isn't very useful, you can
+replicate the app's Postgres database [using GOV.UK Docker][replicate-db].
 
-You will need to have [Mapit](https://github.com/alphagov/mapit) checked out locally,
-and you'll need to [import data from S3](https://github.com/alphagov/mapit/blob/master/import-db-from-s3.sh).
+[replicate-db]: https://github.com/alphagov/govuk-docker/blob/master/docs/how-tos.md#how-to-replicate-data-locally
```
Comment (ChrisBAshton) [inline README.md:29]: We've also been adding this at the end of the section:

```
**Use GOV.UK Docker to run any commands that follow.**
```

**Review by ChrisBAshton — COMMENTED** [review]
Looks good, just a couple of comments for consistency with the other PRs 👍 You may also want to consider removing the hard-wrap newlines throughout the README.

File: README.md (hunk 2 above)
Comment (1pretz1) [reply README.md:29]: Ah cool, I've added this to the section.

**Comment by 1pretz1** [comment]
Cheers for the review @ChrisBAshton! Sure thing, I've updated as per your comments.

As for the wrapped new lines. I personally prefer the lines to be shorter as I think they're easier to work with in general, my editor does this for me. However, if you feel strongly about it I can remove the wrapping?

**Comment by ChrisBAshton** [comment]
> As for the wrapped new lines. I personally prefer the lines to be shorter as I think they're easier to work with in general, my editor does this for me. However, if you feel strongly about it I can remove the wrapping?

Not something I feel hugely strongly on, and there's no official guidance on this in things like the GDS Way, but ISTR it came out of a Slack conversation with @benthorner & others in the devtools channel, which concluded a dislike for hard-wrapping in markdown (but not in commit messages). It's been quietly adopted in a couple of PRs so far:

https://github.com/alphagov/link-checker-api/pull/456/commits/bc55e25667e793af4f92b6b1ee38183642cb1b8d
https://github.com/alphagov/publishing-api/pull/1936/commits/0aef8cda57b8c7910bc50ae6a2af21b600fe9790

**Comment by 1pretz1** [comment]
Ah, okay. Well if it's been actively agreed upon then that's fair enough. I've updated the PR accordingly. 

I'll also have to silence my markdown hints concerning line length! 😆🔥

**Review by ChrisBAshton — APPROVED** [review]
Thought I'd ✅ 'd this already - apologies :)

**Pull request merged**